Tests for setconstrLB! and setconstrUB!

ulfworsoe opened the pull request:
Tests for chaning changing linear constraint bounds. Related to discussion here:
https://github.com/JuliaOpt/Mosek.jl/commit/31941d63a5d1db2c62ff09caff0355fac3ee168b

mlubin commented on test/linproginterface.jl:
```diff
@@ -128,4 +128,79 @@ function linprogsolvertest(solver::AbstractMathProgSolver)
+    loadproblem!(m, [ 1.0 0.0 ; 0.0 1.0 ], [-Inf, -Inf], [Inf,Inf], [1.0, -1.0], [0.0,0.0], [0.0,0.0], :Min)
+
+    optimize!(m)
+    @test_approx_eq getobjval(n) 0.0
```
Typo: ``s/n/m/``

mlubin commented:
Thanks, @ulfworsoe. These tests are actually testing functionality that many solvers (Gurobi, CPLEX) don't support, namely two-sided constraints. It would be better to test updating constraint bounds that don't involve two-sided constraints and don't switch constraint senses.

ulfworsoe commented:
Ah, ok. I see. I can put these in a separate file.

ulfworsoe commented:
@miles, I think this should do the trick. I added an extra function with the ranged-bounds tests. It found two bugs in the Mosek interface that I have fixed in the head revision of Mosek.jl.

mlubin commented:
Thanks!
